ui/ui-components/components/tabs/destination.tsx

```diff
@@ -6,10 +6,19 @@ export default function DestinationTabs({
 }: {
   destination: Models.DestinationType;
 }) {
-  const tabs =
-    process.env.GROUPAROO_UI_EDITION === "enterprise"
-      ? ["edit", "data", "exports"]
-      : ["exports"];
+  let tabs = [];
```

Comment:
I made decisions on tabs throughout this PR. I'm not sure that they're right, but I figured we could easily add them when the time came.